test/onnx/onnx_test_common.py

```diff
@@ -52,6 +58,71 @@ def run_model_test(test_suite: _TestONNXRuntime, *args, **kwargs):
     return verification.verify(*args, options=options, **kwargs)
 
 
+def run_onnx_reference_runtime(
```

Comment:
Any particular reason why we won't want reference runtime test?

Reply:
I do like the reference runtime tests but I feel that the runtime is not robust enough yet. It was added because ort didn't run our tests, but after we fixed errors on our functions ort has been empirically shown to be more robust than the reference runtime.